services/prefect/flows/nyt_updater.py

```diff
@@ -0,0 +1,34 @@
+from prefect import Flow
```

Comment:
`nyt_updater.py` feels too generic.  I wonder if it should be scheduled_nyt_and_parquet_updater.py or something? 

Maybe also add a comment explaining that this is the main updater that is responsible for getting NYT data into a parquet file in time for the snapshot build to pick it up.